samples/matter/light_bulb/configuration/nrf52840dk_nrf52840/pm_static.yml

```diff
@@ -1,38 +1,38 @@
 mcuboot:
     address: 0x0
-    size: 0xc000
+    size: 0x6000
     region: flash_primary
 mcuboot_pad:
-    address: 0xc000
+    address: 0x6000
     size: 0x200
 app:
-    address: 0xc200
-    size: 0xebe00
+    address: 0x6200
+    size: 0xf1e00
 mcuboot_primary:
     orig_span: &id001
         - mcuboot_pad
         - app
     span: *id001
-    address: 0xc000
-    size: 0xec000
+    address: 0x6000
+    size: 0xf2000
     region: flash_primary
 mcuboot_primary_app:
     orig_span: &id002
         - app
     span: *id002
     address: 0xc200
```

Comment:
This seems to be wrong. It should be 0x6200 I guess.

Reply:
This partition span contains mcuboot_pad and app partitions so it should start from 0x6000, shouldn't it? An application partition is started from 0x6200 and this is described in mcuboot_primary_app.

Reply:
It think you meant `mcuboot_primary`, as `mcuboot_primary_app` contains only application part, so it should be located after the mcuboot + padding. And that's why it is shifted by 0x200 compared to the `mcuboot_primary`.